ubuntu-headless-qt/script/apt_wifi_ble.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Check if the script is run as root
+# install after chroot
+export LC_ALL=C
+chmod 777 /tmp
+apt update
+
+# Set DEBIAN_FRONTEND globally
+export DEBIAN_FRONTEND=noninteractive
+
+# Install debconf-utils for preconfiguring
+apt install -y debconf-utils
+
+# time zone data
+echo 'tzdata tzdata/Areas select Asia' | sudo debconf-set-selections
+echo 'tzdata tzdata/Zones/Asia select Ho_Chi_Minh' | sudo debconf-set-selections
```

Comment:
Should we move these variables into a "configuration" file? So Users can set them to whatever they want instead of hardcoding.

Reply:
This script is redundant and will be removed